src/main/steps/common/equality/createToken.ts

```diff
@@ -0,0 +1,25 @@
+import crypto from 'crypto';
```

Comment:
As already said create a controller and have this as private method so its all in one place for better maintenance and understanding.